fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -720,4 +722,16 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    /**
+     * Check if a path has a segment prefixed with fedora:
+     *
+     * @param externalPath the path.
+     */
+    public static void hasRestrictedPath(final String externalPath) {
```

Comment:
It seems like this method should be in FedoraLDP as a `private` method... since that is the only class that uses it.